src/scss/components/heroSlide.scss

```diff
@@ -0,0 +1,249 @@
+
+  *{
+    margin:0;
+    padding: 0;
+  }
+.Hero {
+  overflow: hidden;
+}
+.Hero{
+  height:100vh ;
+}
+.carousel {
+  height: fit-content;
+  display: inline-flex;
+  overflow: hidden;
+  scrollbar-width: none;
+  transform: translateX(0);
+  will-change: transform;
+  transition: transform 0.3s ease-out;
+  cursor: grab;
+  user-select: none;
+  
+}
+
+.hero__slide__container.false{
+  height: fit-content;
+  width: 100vw;
+  display: grid;
+  flex-direction: column;
+  grid-template-columns: repeat(14,5.4%) ;
+  grid-column-gap:1.85%;
+  align-items: flex-start;
+}
+.hero__slide__container.true{
+  height: 100%;
+  width: 100vw;
+  display: grid;
+  flex-direction: column;
+  grid-template-columns: repeat(14,5.4%) ;
+  grid-template-rows: 50% 50%;
+  grid-column-gap:1.85%;
+  align-items: flex-end;
+}
+.carousel .false .hero__img{
+  grid-column: 2/14;
+}
+.carousel .true .hero__img{
+  grid-column: 1/15;
+  max-height: 617px;
+  grid-row: 1/3;
+  
+}
+.hero__slide__container img{
+  width: 100%;
+  height: auto;
+}
+.carousel .false img{
+  margin-top: 40px;
+}
+
+.grabbing {
+  cursor: grabbing;
+}
+.carousel .hero__slide__container.false .info {
+  grid-column: 2/14;
+  width: auto;
+  justify-content: center;
+}
+.carousel .hero__slide__container.true .info {
+  grid-column: 2/14;
+  width: auto;
+  justify-content: center;
+  align-items: flex-end;
+  grid-row: 2;
+}
+
+.carousel .hero__slide__container .info h1{
```

Comment:
use the class that you assigned to the element `.hero__title` instead of the `h1` tag